sdk/attestation/azure-security-attestation/azure/security/attestation/aio/_client_async.py

```diff
@@ -22,73 +16,87 @@
+        an exception.
+    :paramtype validation_callback: ~typing.Callable[[~azure.security.attestation.AttestationToken, ~azure.security.attestation.AttestationSigner], None]
+    :keyword bool validate_signature: If True, validate the signature of the token being validated.
+    :keyword bool validate_expiration: If True, validate the expiration time of the token being validated.
+    :keyword str issuer: Expected issuer, used if `validate_issuer` is true.
+    :keyword float validation_slack: Slack time for validation - tolerance applied
+        to help account for clock drift between the issuer and the current machine.
+    :keyword bool validate_issuer: If True, validate that the issuer of the token matches the expected issuer.
+    :keyword bool validate_not_before_time: If true, validate the "Not Before" time in the token.
+
+    .. tip::
+        The `validate_token`, `validation_callback`, `validate_signature`,
+        `validate_expiration`, `validate_not_before_time`, `validate_issuer`, and
+        `issuer` keyword arguments are default values applied to each API call within
+        the :py:class:`AttestationClient` class. These values can be
+        overridden on individual API calls as needed.
+
+    For additional client creation configuration options, please see `Python Request
+    Options <https://aka.ms/azsdk/python/options>`_.
 
     """
+
     def __init__(
-        self,
-        credential,  # type: 'AsyncTokenCredential'
-        instance_url, #type: str
-        **kwargs #type: Any
-    ): #type: (...) -> None
+        self, endpoint: str, credential: "AsyncTokenCredential", **kwargs: Any
+    ) -> None:
         if not credential:
             raise ValueError("Missing credential.")
-        self._config = AttestationClientConfiguration(credential, instance_url, **kwargs)
-        self._client = AzureAttestationRestClient(credential, instance_url, **kwargs)
+        self._config = AttestationClientConfiguration(**kwargs)
+        self._client = AzureAttestationRestClient(credential, endpoint, **kwargs)
         self._statelock = Lock()
         self._signing_certificates = None
 
     @distributed_trace_async
-    async def get_openidmetadata(
-        self, 
-        **kwargs #type: Any
-        ): #type: (...) -> Any
-        """ Retrieves the OpenID metadata configuration document for this attestation instance.
-
-        :return: OpenId Metadata document for the attestation service instance.
-        :rtype: Any
+    async def get_open_id_metadata(self, **kwargs: Any) -> Dict[str, Any]:
+        """Retrieves the OpenID metadata configuration document for this attestation instance.
+
+        The metadata configuration document is defined in the `OpenID Connect
+        Discovery <https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationResponse>`
+        specification.
+
+        The attestation service currently returns the following fields:
+        * issuer
```

Comment:
So that the bulleted list renders correctly:
```suggestion
        The attestation service currently returns the following fields:

        * issuer
```